### PR TITLE
RTPS test branch

### PR DESCRIPTION
**Pull request by dagar**
Old test branch I was going to delete, but looks safe to merge for anyone that might want to play with RTPS.

### CODE REVIEW

**Comment by dagar** [comment]
Feel free to merge or delete.

File: ROMFS/px4fmu_common/init.d/rcS [hunk 1]
```diff
@@ -670,6 +670,10 @@ then
 	then
 		mavlink start -d ${MAVLINK_COMPANION_DEVICE} -b 115200 -m minimal -r 1000
 	fi
+	if param compare SYS_COMPANION 6460800
```
Comment (TSC21) [inline ROMFS/px4fmu_common/init.d/rcS]: Any reason to establish the RTPS client to half the baud rate of the ESP8266?

Comment (dagar) [reply ROMFS/px4fmu_common/init.d/rcS]: I was following another example at the time using this baudrate. Feel free to change it to a more appropriate default, or add a range of desired rates.

**Comment by dagar** [comment]
This is safe to merge. SYS_COMPANION options can be expanded as needed.

**Pull request merged**